pkg/config/config.go

```diff
@@ -74,8 +74,8 @@ var (
 			Required:   true,
 		},
 		{
-			Name:     "exclude_nodes_by_label",
-			Help:     "exclude jenkins nodes by label",
+			Name:     "node_with_label",
```

Comment:
Should be in plural, I had a typo in my suggestion

```suggestion
			Name:     "nodes_with_label",
```